src/lib.rs

```diff
@@ -43,36 +36,32 @@ extern crate serde_test;
 #[cfg(test)]
 extern crate itertools;
 
-mod errors;
-mod traits;
-mod digit;
-mod bitwidth;
-mod bitpos;
-mod storage;
-mod radix;
-mod apint;
-mod digit_seq;
-mod checks;
-mod uint;
-mod int;
-mod utils;
+//NOTE: the file structure used in this library has less to do with the actual dependencies between
+//files and more about organizing files in a way that helps with programmers finding where stuff is.
 
-pub use apint::{
-    ApInt,
-    ShiftAmount
-};
-pub use errors::{
+//The `ApInt` definition and most of the extremely unsafe function impls on `ApInt`s are located in
```

Comment:
```suggestion
// The `ApInt` definition and most of the extremely unsafe function impls on `ApInt`s are located in
```

Reply:
I have always used no spaces between the `//` and the start of the sentences.

Reply:
I have decided to add in all the spaces.